components/kubeflow/launcher/kubeflow_tfjob_launcher_op.py

```diff
@@ -14,17 +14,31 @@
 
 from kfp import dsl
 
-def kubeflow_tfjob_launcher_op(container_image, command, number_of_workers: int, number_of_parameter_servers: int, tfjob_timeout_minutes: int, output_dir=None, step_name='TFJob-launcher'):
+def kubeflow_tfjob_launcher_op(container_image,
+                               command,
+                               version,
+                               pvc_map_str,
+                               delete_tfjob_after_done,
+                               has_master: bool,
+                               number_of_workers: int,
+                               number_of_parameter_servers: int,
+                               tfjob_timeout_minutes: int,
+                               output_dir=None,
+                               step_name='TFJob-launcher'):
     return dsl.ContainerOp(
         name = step_name,
-        image = 'gcr.io/ml-pipeline/ml-pipeline-kubeflow-tf:1d55a27cf8b69696f3ab5c10687edf2fde0068c7',
+        image = 'liuhougangxa/tfjob-launcher',
```

Comment:
Is this intentional?

Reply:
yes, in fact kubeflow_tfjob_launcher_op.py is just used as an example about what tfjob-launcher  containerOP should be
when image updated in gcr.io/ml-pipeline, the image will be updated, too